package util.exception;

/**
 * Represent an error occurs when the user does not add the content of the task
 */
public class NoTaskException extends Exception {

}
